addons/point_of_sale/static/src/app/screens/product_screen/product_list/product_list.js

```diff
@@ -44,6 +44,19 @@ export class ProductsWidget extends Component {
             observer.observe(productsWidget);
             return () => observer.disconnect();
         });
+        const toggleIsMobile = () => this.toggleIsMobile();
+        onMounted(() => {
+            toggleIsMobile();
+            window.addEventListener("resize", toggleIsMobile);
+        });
+        onWillUnmount(() => {
+            window.removeEventListener("resize", toggleIsMobile);
+        });
+    }
+
+    toggleIsMobile() {
+        const element = this.productsWidgetRef.el;
+        this.state.isMobile = (element && element.offsetWidth < 350) || this.ui.isSmall;
```

Comment:
May I know why `ui.isSmall` ~~is~~isn't enough as the condition to add the 'scrollbar-hidden' class?

Reply:
It is indeed enough, made the changes